web/themes/custom/pineapple/Gruntfile.js

```diff
@@ -29,7 +29,8 @@ module.exports = function(grunt) {
           'sass/core/__core.scss': 'sass/core/**/*.scss',
           'sass/base/__base.scss': 'sass/base/**/*.scss',
           'sass/layouts/__layouts.scss': 'sass/layouts/**/*.scss',
-          'sass/components/__components.scss': 'sass/components/**/*.scss'
+          'sass/components/__components.scss': 'sass/components/**/*.scss',
+          'sass/components/field/__field.scss': 'sass/field/**/*.scss'
```

Comment:
This is not necessary because the field file and directory is already added in the components file

Reply:
done